README.md

```diff
@@ -132,14 +122,28 @@ Pick one!
 
 - Pre-built environment, in cloud with GitHub Codespaces, described in the [Quickstart](#quickstart-with-codespaces).
 - Pre-built environment, on device with Docker Desktop
-- Manual setup environment, on device with .NET and NuGet installed
+- Manually setup environment, on device with .NET and NuGet installed
 
 The first approach is recommended for minimal user effort in startup and maintenance. 
 The third approach will require you to manually update or maintain your local environment, to reflect any future updates to the repo.
 
-To setup the development environment you can leverage either GitHub Codespaces, a local .NET with NuGet environment (using Anaconda or venv), or a VS Code Dev Container environment (using Docker).
+#### 1.1: Quickstart with Codespaces
```

Comment:
I did split the last readme because Natalia and Kristen asked for me to put more emphasis on the Codespaces experience.

Reply:
Makes sense - I moved it down because in the flow it felt a bit awkward to have a quick start, but then see pre-requisites, which you needed at the start anyway.